src/Components/VertNav/VertNav.js

```diff
@@ -24,8 +44,11 @@ export default function VertNav(props) {
         onUpdate={e => {
           if (!e)
             setHidden(true);
-          else
+          else {
+            if (props.callbacks && props.callbacks[e.id])
+              props.callbacks[e.id](e);
             setHidden(false);
+          }
```

Comment:
This is the only change of code in this PR. It allows for optional functions passed through props to be called on update of the scrollspy component.